apps/src/lab2/levelEditors/validations/EditValidation.tsx

```diff
@@ -94,6 +94,22 @@ const EditValidation: React.FunctionComponent<EditValidationProps> = ({
           }}
         />
       </div>
+      <div className={moduleStyles.row}>
+        <label htmlFor="callout" className={moduleStyles.label}>
+          Callout (optional):
+        </label>
+        <textarea
+          id="callout"
+          name="callout"
+          rows={1}
+          className={moduleStyles.callout}
+          value={validation.callout}
+          onChange={e => {
+            validation.callout = e.target.value;
```

Comment:
Modifying props object seems tricky to me, are we sure it'll always work as expected? (Keeping in mind possible react/js magic)
Maybe rewrite like this?
`e => onValidationChange({...validation, callout: e.target.value})`

Reply:
Oh, good question.  I've left as-is for now to be consistent with the rest of the file, but we could consider updating all of them separately.